release-team/role-handbooks/communications/README.md

```diff
@@ -61,11 +61,11 @@ Throughout the release cycle, the main Communications deliverables include:
 
 ### Collect Major Themes
 
-A GitHub Discussion must be open to invite all SIG leads and members to add Major Themes for inclusion in the Release Blog and Release Notes. The discussion must be opened in kubernetes/sig-release under General Category. Past discussions: [1.25](https://github.com/kubernetes/sig-release/pull/1987), [1.24](https://github.com/kubernetes/sig-release/discussions/1868), [1.23](https://github.com/kubernetes/sig-release/discussions/1709), [1.22](https://github.com/kubernetes/sig-release/discussions/1575), [1.21](https://github.com/kubernetes/sig-release/discussions/1436).
+A GitHub Discussion must be open to invite all SIG leads and members to add Major Themes for inclusion in the Release Blog and Release Notes. The discussion must be opened in kubernetes/sig-release under General Category. Past discussions: [1.31](https://github.com/kubernetes/sig-release/discussions/2526), [1.25](https://github.com/kubernetes/sig-release/pull/1987), [1.24](https://github.com/kubernetes/sig-release/discussions/1868), [1.23](https://github.com/kubernetes/sig-release/discussions/1709), [1.22](https://github.com/kubernetes/sig-release/discussions/1575), [1.21](https://github.com/kubernetes/sig-release/discussions/1436). 
 
-In the case of a low reponse rate in the Github discussion, each SIG should be pinged via their individual Slack channels and the #chairs-and-techleads channel.
+Reach out to each SIG in their slack channels and  #chairs-and-techleads channel to bring attention to the Major Themes discussion around Enhancements Freeze. To help coordinate outreach to all the SIGs, you can use a issue in the kubernetes/sig-release repe. Past issues: [1.31](https://github.com/kubernetes/sig-release/issues/2527). In the case of a low reponse rate in the Github discussion, do a round of reminders in Slack about 2 weeks before the Major Themes deadline.
```

Comment:
What if we designated one of the release meetings (2 weeks from Major theme deadline) to have a small round table where we would define Major Themes? A bit like how it happened with this release.

I think making it an official appointment would be the best way to handle it, wdyt?

Reply:
We're making changes to the major themes process. I'm opening a separate PR for that.